receiver/awscontainerinsightreceiver/internal/neuron/metric_name.go

```diff
@@ -0,0 +1,18 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package neuron
+
+const (
+	NeuronCoreUtilization                       = "neuroncore_utilization_ratio"
+	NeuronCoreMemoryUtilizationConstants        = "neuroncore_memory_usage_constants"
+	NeuronCoreMemoryUtilizationModelCode        = "neuroncore_memory_usage_model_code"
+	NeuronCoreMemoryUtilizationSharedScratchpad = "neuroncore_memory_usage_model_shared_scratchpad"
+	NeuronCoreMemoryUtilizationRuntimeMemory    = "neuroncore_memory_usage_runtime_memory"
+	NeuronCoreMemoryUtilizationTensors          = "neuroncore_memory_usage_tensors"
+	NeuronDeviceHardwareEccEvents               = "neurondevice_hw_ecc_events_total"
```

Comment:
where is this being used?

Reply:
These are used in the Decorator, to populate empty metrics in case of no workload running